contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -64,7 +65,7 @@ GRANT SELECT ON sys.sysdatabases TO PUBLIC;
 
 -- PG_NAMESPACE_EXT
 CREATE VIEW sys.pg_namespace_ext AS
-SELECT BASE.* , DB.name as dbname FROM
+SELECT BASE.* , DB.orig_name as dbname FROM
```

Comment:
This internal view is being used by many other system objects. Are we sure that we need orig_name here?

Reply:
All system objects shall now return case sensitive catalog name, I think what Anju did is right.
Since user queries like select * from information_schema.tables where TABLE_CATALOG = db_name() will stop working for case sensitive db names (though this query does not makes sense but is still valid).
By the above change, all objects consuming ``sys.pg_namespace_ext`` to fetch orig_name will get the case sensitive names.